Makefile

```diff
@@ -149,6 +149,20 @@ vm-local-repos: vm
 		--run-command "systemctl enable lorax-composer" \
 		$(TEST_OS)
 
+vm-releng:
```

Comment:
That should depend on the `vm` target or no ?

Reply:
If this is the target intended for usage downstream then `vm-local-repos` should be removed, right ?

Reply:
The process of creating & customizing the rel-eng image is a bit different from fedora/nightly images. The first step is to manually create an image using `bots/image-create-rhel-compose` script, together with environment variables determining URLs to the base and extras compose (yes, it can default to -latest, but will likely fail due to the changes in URL structures we've seen lately). The second step is to run `make vm-releng`, which customizes the image in an analogous way to what we've used for the stock bots images.

However, if you think I should change that workflow to be aligned with `vm` target, I'd propose creating a new target (`vm-releng-image`) that would check for presence of the necessary env variables and then would fire up the rel-eng creation script. This target could then be a dependency of `vm-releng` target.

As for removal of `vm-local-repos`, I don't have a strong opinion about it. It purpose is/was a bit different, but on the other hand it will likely find no more use when the `vm-releng` target is introduced, I guess.

Reply:
> The process of creating & customizing the rel-eng image is a bit different from fedora/nightly images. 

Then you need to add a few lines to `test/README` to share the knowledge or link to the bots repo, etc.

> However, if you think I should change that workflow to be aligned with vm target, I'd propose creating a new target (vm-releng-image) 

There is a single-use case here from what I can see. We don't need any more make targets. Any related changes should go into the current make target depending on how you decide to evolve this PR.

> As for removal of vm-local-repos, I don't have a strong opinion about it. It purpose is/was a bit different

It's a very similar make target which I believe is unused ATM. In the case that you are adding a new target either remove the previous one or maybe try merging the implementation so 1 target can do 2 very closely related things.

Reply:
OK, thanks for you insights. I decided to remove the `vm-local-repos` target altogether. I think that this target and the new one do two different things (but to achieve mostly the same goal) and it would be quite cumbersome to merge them together in a seamless manner.